### DISCUSSION TITLE
Integrate draw.io templates into LeanIX

### DISCUSSION THREAD

**Discussion by skyphab2000** [SAP/btp-solution-diagrams#22] [Ideas]
Hi all
The templates are a great set of features! It would be very helpful to have the draw.io templates integrated into LeanIX (diagramming is also based on draw.io) so that there is one central place to store those nice looking overviews for SAP-landscapes and make it available within the company.

**Comment by fabianleh** [comment]
Hi @skyphab2000 ,

sorry for the late reply, in regards to the LeanIX topic.
We already discussing this topic with LeanIX, what you can do already (I guess you are aware of this) is to import the BTP Solution Diagrams into "free draw" and link the fact sheets etc.  best,
Fabian
